README.md

```diff
@@ -193,19 +194,32 @@ For most users, setting `image: [path-to-image]` on a per-page basis should be e
 * `path` - The relative path to the image. Same as `image: [path-to-image]`
 * `twitter` - The relative path to a Twitter-specific image.
 * `facebook` - The relative path to a Facebook-specific image.
-* `height` - The height of the Facebook (`og:image`) image
-* `width` - The width of the Facebook (`og:image`) image
+* `height` - The height of the Facebook (`og:image`) image and JSON-LD image object.
+* `width` - The width of the Facebook (`og:image`) image and JSON-LD image object.
+
+The JSON-LD will default to the `image: ` tag unless `path: ` `height: ` and `width: `.
 
 You can use any of the above, optional properties, like so:
 
 ```yml
 image:
+  path: /img/banner.png
```

Comment:
I put path in here because it wasn't clear if I used "facebook" as a url if it would count as the `url` for the image in JSON.